ts/features/idpay/wallet/components/IDPayInitiativesListComponents.tsx

```diff
@@ -52,13 +53,26 @@ export const IDPayInitiativesList = ({
 type ListItemProps = {
   item: InitiativesStatusDTO;
   idWallet: string;
+  isLast: boolean;
 };
 
-export const IDPayInitiativeListItem = ({ item, idWallet }: ListItemProps) => {
+export const IDPayInitiativeListItem = ({
+  item,
+  idWallet,
+  isLast
+}: ListItemProps) => {
   const { initiativeName } = item;
 
   return (
-    <NBlistItem style={{ justifyContent: "space-between", paddingRight: 0 }}>
+    <NBlistItem
+      itemDivider={false}
+      last={isLast}
+      style={{
+        justifyContent: "space-between",
+        paddingRight: 0,
+        paddingLeft: 0 // required to override NBlistItem default padding
+      }}
+    >
```

Comment:
Can we refactor this item to remove NB dependency?

Reply:
we'd need to either remake it or make a new ListItem that uses the RemoteSwitch; I don't think it's in this pr's scope, should I add a new DS debt task?

Reply:
Yes, a new task should be the right choice.

Reply:
added 👍